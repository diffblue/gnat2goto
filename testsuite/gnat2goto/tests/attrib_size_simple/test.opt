ALL XFAIL cbmc fails with "reason unimplemented"

