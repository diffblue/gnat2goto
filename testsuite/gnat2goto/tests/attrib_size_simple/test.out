[good_size.assertion.2] line 25 assertion I = 8: SUCCESS
[good_size.assertion.1] line 28 Ada Check assertion: FAILURE
[good_size.assertion.3] line 29 assertion I = 8: SUCCESS
[good_size.assertion.4] line 35 assertion J = 4: SUCCESS
[good_size.assertion.5] line 42 assertion J = 8: SUCCESS
[good_size.assertion.6] line 46 assertion K = 32: SUCCESS
[good_size.assertion.7] line 49 assertion K = 32: SUCCESS
[good_size.assertion.8] line 53 assertion R'Size = 33: SUCCESS
[good_size.assertion.9] line 57 assertion R_Obj'Size >= R'Size: SUCCESS
VERIFICATION FAILED
